Misc/NEWS.d/next/Core and Builtins/2020-07-19-23-54-52.bpo-41343.IR0Zy-.rst

```diff
@@ -0,0 +1,2 @@
+Convert methods of :class:`complex` to Argument Clinic. Patch by Dong-hee
```

Comment:
We usually do not add NEWS entries when convert code to Argument Clinic. And the performance effect is insignificant in this case.